R/pkg/NAMESPACE

```diff
@@ -287,6 +287,7 @@ export("as.DataFrame",
        "read.json",
        "read.parquet",
        "read.text",
+       "sparkLapply",
```

Comment:
Yeah I think lets stick to `sparkr.lapply` or `spark.lapply`. As we saw in SPARK-12148 - overloading names can cause unforeseen conflicts.